pom.xml

```diff
@@ -50,6 +50,7 @@
     <sparkshim.artifactId>spark-sql-columnar-shims-spark32</sparkshim.artifactId>
     <celeborn.version>0.3.0-incubating</celeborn.version>
     <arrow.version>14.0.1</arrow.version>
+    <uniffle.version>0.8.0</uniffle.version>
```

Comment:
Does Uniffle `0.8.0` have requirements for Spark version?

Reply:
Only `spark.major.version` matters, it's defined already in gluten